Content.Shared/Chat/V2/Components/CanEmoteComponent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Robust.Shared.GameStates;
```

Comment:
These components are all taken from the original ChatFactor PR for the time being. The data in them is somewhat unimportant with the exception of the channels data in the radio events.